src/content/en/2024/javascript.md

```diff
@@ -1,12 +1,13 @@
 ---
+
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: JavaScript
 description: JavaScript chapter of the 2024 Web Almanac covering the usage of JavaScript on the web, libraries and frameworks, compression, web components, and source maps.
 hero_alt: Hero image of the Web Almanac characters cycling to power a website.
-authors: []
-reviewers: []
-editors: []
-analysts: []
```

Comment:
@hadiamjad , @nrllh please confirm if I've missed anyone off of this. There were other authors, reviewers, and editors mentioned in #3592 and know @NishuGoel was tagged in the chat, but not sure if any of them contributed in the end?